rskj-core/src/test/java/co/rsk/peg/bitcoin/PocSighashTest.java

```diff
@@ -0,0 +1,776 @@
56c8fa76568401004cda64522103462ab7041341dadd996dc12ef0c118ca8ccb546498cbf304f7ffe0f1b12f9a9e210362634ab57dae9cb373a5d536e66a8c4f67468bbcfb063809bab643072d78a1242103c5946b3fbae03a654237da863c9ed534e0878657175b132b8ca630f245df04db53ae6702d002b2755221029cecea902067992d52c38b28bf0bb2345bda9b21eca76b16a17c477a64e433012103284178e5fbcc63c54c3b38e3ef88adf2da6c526313650041b0ef955763634ebd2103b9fc46657cf72a1afa007ecf431de1cd27ff5cc8829fa625b66ca47b967e6b2453ae68ffffffff6a040fa1ec01bac865d4802083a435fef32052091d6d7bd02cc784d00d75e84000000000fd6e010047304402206a00bb7a876ecbacd13c06a3e8ca3c55d022e9f35ad6211c65bc4ab3e2a12f1b022051a1d0861a5e812c6c2ffa1be0467fe953ceff6ecb9fc3eb68843441d5c3d7a501473044022044744cceb33a776928c61c8d01cf3acf58c7159ccd5a69914cb78be2a59d5b2d022020ff71156583ba563e9a4d6a8f0bd463d53a6612e831082e0ac9c1c2d1a6d79101004cda64522103462ab7041341dadd996dc12ef0c118ca8ccb546498cbf304f7ffe0f1b12f9a9e210362634ab57dae9cb373a5d536e66a8c4f67468bbcfb063809bab643072d78a1242103c5946b3fbae03a654237da863c9ed534e0878657175b132b8ca630f245df04db53ae6702d002b2755221029cecea902067992d52c38b28bf0bb2345bda9b21eca76b16a17c477a64e433012103284178e5fbcc63c54c3b38e3ef88adf2da6c526313650041b0ef955763634ebd2103b9fc46657cf72a1afa007ecf431de1cd27ff5cc8829fa625b66ca47b967e6b2453ae68ffffffff114c09191b5b1fe685640c23a2ab95bfec521d49ddde4bb96032471762b441e400000000fd6f0100483045022100979a3c231a8ca8c1aa081dffe6cbe0d7ad5987ce4329757cfd8e0c3bfbc5ba8c02202357deaf74162fefc1a05ddf4c2bb2b7054fa105ceddf6ed4e061dab88ae056f0147304402204ebb79cbc3fd9260e28b1f9c4564f3352ce3adba6dc390279808d42961152c9a022034c215c5e9fc88b1622d0636950b5dd1de7b1322e9857736f9a3a63e9bb2cdad01004cda64522103462ab7041341dadd996dc12ef0c118ca8ccb546498cbf304f7ffe0f1b12f9a9e210362634ab57dae9cb373a5d536e66a8c4f67468bbcfb063809bab643072d78a1242103c5946b3fbae03a654237da863c9ed534e0878657175b132b8ca630f245df04db53ae6702d002b2755221029cecea902067992d52c38b28bf0bb2345bda9b21eca76b16a17c477a64e433012103284178e5fbcc63c54c3b38e3ef88adf2da6c526313650041b0ef955763634ebd2103b9fc46657cf72a1afa007ecf431de1cd27ff5cc8829fa625b66ca47b967e6b2453ae68ffffffffefd39f04aa64aec104c68bbecef06dc8ffa916cec5c536b8e03a19e40b2f9af101000000fd700100483045022100a668d635ca8528be0b0affc22c64636f2128b1a1a0891a5f3d2c2579eb9ab4ea0220450088cb20042ce52ebd32c61f2d3b5085c6cd725c36ffae1a751878091a81b601483045022100e25e9ee15ac340063a27258ca905c273cd6fa907271c3d5ddcf1daea2969ddc6022059f190175e98636708bbc4c9cb9809216ee09a0f0d2794e9bf87220567e43c7401004cda64522103462ab7041341dadd996dc12ef0c118ca8ccb546498cbf304f7ffe0f1b12f9a9e210362634ab57dae9cb373a5d536e66a8c4f67468bbcfb063809bab643072d78a1242103c5946b3fbae03a654237da863c9ed534e0878657175b132b8ca630f245df04db53ae6702d002b2755221029cecea902067992d52c38b28bf0bb2345bda9b21eca76b16a17c477a64e433012103284178e5fbcc63c54c3b38e3ef88adf2da6c526313650041b0ef955763634ebd2103b9fc46657cf72a1afa007ecf431de1cd27ff5cc8829fa625b66ca47b967e6b2453ae68ffffffff0124e710000000000017a9140e39a099dabb03863ae5484943896a7c9eb9f9dd8700000000";
```

Comment:
Where did you get this raw tx hex? devnet, testnet, mainnet?

Reply:
Same here.